src/app/page.module.scss

```diff
@@ -8,9 +9,13 @@
   gap: $spacing-m;
   max-width: $max-width-main;
   width: 100%;
-  padding: $spacing-xl;
+  padding: $spacing-m;
 
   h2 {
     margin-bottom: $spacing-m;
   }
+
+  @include from-tablet {
```

Comment:
Please move this mixin a bit up so it will be immediately after the 
` width: 100%;
  padding: $spacing-m;`